emain/emain-window.ts

```diff
@@ -699,9 +699,25 @@ ipcMain.on("delete-workspace", (event, workspaceId) => {
             console.log("user cancelled workspace delete", workspaceId, ww?.waveWindowId);
             return;
         }
+        
+        if (workspaceList.length > 1 && ww?.workspaceId == workspaceId){
+            
+            const workspaceIdindex =  workspaceList.findIndex(wse => wse.workspaceid === workspaceId);
+                
+            const moveIndex = (workspaceIdindex > 0) ? workspaceIdindex - 1 : workspaceIdindex + 1;   
+            const prevNextWorkspace = workspaceList[moveIndex]
+            
+            await ww.switchWorkspace(prevNextWorkspace.workspaceid);
+			// this does it faster without restarting the window
+            ww.activeTabView.webContents.reloadIgnoringCache(); // the blank window bug from 0.10.1 OCCASIONALLY occurs if this is not here,  ( issue #1567 )
```

Comment:
Is this still happening in the latest release?

Reply:
![image](https://github.com/user-attachments/assets/daf2d8f2-e85f-4422-a1cb-9343ded9c8fd)
been doing some testing on
![image](https://github.com/user-attachments/assets/d6b6f841-182a-4fe4-89e3-4678cdeb6fbf)

I synced and pulled locally and thereafter tested it again without the  ww.activeTabView.webContents.reloadIgnoringCache();, still happens from time to time without that line.

Reply:
is this specifically in your branch where you notice this happening or do you notice this in the main version of the app too?

Reply:
As of right now the only difference with the main version and my branch implementation is literally this, either my use of  the existing code is wrong or i've just found another way to cause the bug when switching workspaces. I noticed that and found a way to avoid it since I have absolutely no clue what's causing it.

Reply:
Left another comment, I think I know why this is happening but the only way to be sure it won't is to do the logic on the backend